requirements.txt

```diff
@@ -0,0 +1,4 @@
+# poetry not supported by cloud.gov?
```

Comment:
What is the context on this? Poetry can export to a `requirements.txt` if need be with something like `poetry export --without-hashes --format=requirements.txt > requirements-poetry.txt`.

Reply:
That's a useful command.  We'll definitely need it in the repo somewhere.  We thought that cloud.gov would be able to recognize the poetry file, but apparently it only deals with `requirements.txt`.  So we can generate it with that command.  @Jin-Sun-tts Do you want to run it for this repo and add it to the code someplace?